src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class Reader {
```

Comment:
```suggestion
public class CsvFileReader {
```
Your either make some hierarchy Like Reader -> FileReader -> CsvFileReader or just make it simple by naming it CsvFileReader